openmc/lattice.py

```diff
@@ -517,6 +536,15 @@ def indices(self):
                 :self.shape[2], :self.shape[1], :self.shape[0]]))
 
     @property
+    def _natural_indices(self):
+        if self.ndim == 2:
```

Comment:
Could you comment on this property's value and intended use both here and below?